draft-ietf-quic-multipath.md

```diff
@@ -807,10 +812,10 @@ using multiple packet number spaces.
    1-RTT[0]: DCID=S1, PATH_CHALLENGE[X] -->
```

Comment:
I can't add a suggestion a few line above, but I would state that there are DCIDs C0 and S0, both belonging to Path ID 0, and showing them (like the packets containing MP_NEW_CONNECTION_ID are probably sent using these DCIDs).